### PR TITLE
Add CI testing on travis and appveyor, through tox

### PR DESCRIPTION
**Pull request by jcb91**
Pretty basic at the moment, as there aren't many tests.

### CODE REVIEW

**Comment by jcb91** [comment]
Also not that informative given the current installation method. Once #633 is merged, more tests can make sense.

**Comment by jcb91** [comment]
(effectively this is another part of #626)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+script:
+  - tox -v -e ${TOXENV}
+after_script:
+  # test if TOXENV not in set
+  # see http://unix.stackexchange.com/a/111518
+  - 'if ! [[ ${TOXENV} =~ ^(appveyorartifacts|check)$ ]]; then tox -e coveralls,codecov; fi'
+after_failure:
+  - more .tox/log/* | cat
+  - more .tox/*/log/* | cat
+after_success:
```
Comment (janschulz) [inline .travis.yml]: empty success clause -> Just for reference?

Comment (jcb91) [inline .travis.yml]: Yep. Could be removed, I guess

**Comment by jcb91** [comment]
(tests are currently failing due to some more restructuring that needs adding to pippify PR)

**Comment by jcb91** [comment]
closing as outdated, will open a new PR once I've fixed the installation scripts

**Pull request closed without merging**